aws_lambda_powertools/event_handler/api_gateway.py

```diff
@@ -731,3 +731,84 @@ def register_route(func: Callable):
             self._routes[(rule, methods, cors, compress, cache_control)] = func
 
         return register_route
+
+
+class APIGatewayProxyEventResolver(ApiGatewayResolver):
```

Comment:
```suggestion
class APIGatewayRestResolver(ApiGatewayResolver):
```

Shorter and more explicit name (discussion on Slack too).